CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Allow `xarray_kwargs` for :py:class:`~intake_thredds.THREDDSMergedSource` [#32](https://github.com/NCAR/intake-thredds/pull/32) ([@aaronspring](https://github.com/aaronspring))
+- Allow `concat_kwargs` for :py:class:`~intake_thredds.THREDDSMergedSource` [#34](https://github.com/NCAR/intake-thredds/pull/34) ([@aaronspring](https://github.com/aaronspring))
```

Comment:
I usually generate the changelog via https://github-activity.readthedocs.io/en/latest/. This is fine too. I just wanted to let you know that updating the changelog isn't required :) ...